packages/launch-config/src/debug-config/config.ts

```diff
@@ -56,13 +55,13 @@ function createLaunchConfig(
 /**
  * Configures the launch.json file based on provided options.
  *
+ * @param rootFolder
```

Comment:
Please add description for property `rootFolder`. Is it the project root, the workspace root, etc.